product_docs/docs/pem/8/supported_platforms.mdx

```diff
@@ -10,12 +10,12 @@ For information about the platforms and versions supported by PEM, see [Platform
 
 ## Database compatibility
 
-The following table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD) for PEM 8.x.
+This table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
```

Comment:
Add " for PEM 8.x" at the last of intro sentence  and remove from the table header

Reply:
Done